configs/centerpoint/centerpoint_02pillar_second_secfpn_4x8_cyclic_20e_nus_novelo.py

```diff
@@ -0,0 +1,185 @@
+_base_ = [
+    '../_base_/datasets/nus-3d.py',
+    '../_base_/models/centerpoint_02pillar_second_secfpn_nus.py',
+    '../_base_/schedules/cyclic_20e.py', '../_base_/default_runtime.py'
+]
+
+# If point cloud range is changed, the models should also change their point
+# cloud range accordingly
+point_cloud_range = [-51.2, -51.2, -5.0, 51.2, 51.2, 3.0]
+# For nuScenes we usually do 10-class detection
+class_names = [
+    'car', 'truck', 'construction_vehicle', 'bus', 'trailer', 'barrier',
+    'motorcycle', 'bicycle', 'pedestrian', 'traffic_cone'
+]
+
+model = dict(
+    pts_voxel_layer=dict(point_cloud_range=point_cloud_range),
+    pts_voxel_encoder=dict(point_cloud_range=point_cloud_range),
+    pts_bbox_head=dict(
+        bbox_coder=dict(pc_range=point_cloud_range[:2], code_size=7),
+        common_heads=dict(
+            reg=(2, 2), height=(1, 2), dim=(3, 2), rot=(2, 2), _delete_=True)),
+    # model training and testing settings
+    train_cfg=dict(
+        pts=dict(
+            point_cloud_range=point_cloud_range,
+            code_weights=[1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0, 1.0])),
+    test_cfg=dict(pts=dict(pc_range=point_cloud_range[:2])))
+
+dataset_type = 'NuScenesDataset'
+data_root = 'data/nuscenes/'
+file_client_args = dict(backend='disk')
+
+# db_sampler = dict(
+#     data_root=data_root,
+#     info_path=data_root + 'nuscenes_dbinfos_train.pkl',
+#     rate=1.0,
+#     prepare=dict(
+#         filter_by_difficulty=[-1],
+#         filter_by_min_points=dict(
+#             car=5,
+#             truck=5,
+#             bus=5,
+#             trailer=5,
+#             construction_vehicle=5,
+#             traffic_cone=5,
+#             barrier=5,
+#             motorcycle=5,
+#             bicycle=5,
+#             pedestrian=5)),
+#     classes=class_names,
+#     sample_groups=dict(
+#         car=2,
+#         truck=3,
+#         construction_vehicle=7,
+#         bus=4,
+#         trailer=6,
+#         barrier=2,
+#         motorcycle=6,
+#         bicycle=6,
+#         pedestrian=2,
+#         traffic_cone=2),
+#     points_loader=dict(
+#         type='LoadPointsFromFile',
+#         coord_type='LIDAR',
+#         load_dim=5,
+#         use_dim=[0, 1, 2, 3, 4],
+#         file_client_args=file_client_args))
+
+train_pipeline = [
+    dict(
+        type='LoadPointsFromFile',
+        coord_type='LIDAR',
+        load_dim=5,
+        use_dim=5,
+        file_client_args=file_client_args),
+    dict(
+        type='LoadPointsFromMultiSweeps',
+        sweeps_num=9,
+        use_dim=[0, 1, 2, 3, 4],
+        file_client_args=file_client_args,
+        pad_empty_sweeps=True,
+        remove_close=True),
+    dict(type='LoadAnnotations3D', with_bbox_3d=True, with_label_3d=True),
+    # dict(type='ObjectSample', db_sampler=db_sampler),
```

Comment:
Why do we remove this sampler? If necessary, can directly remove related lines.

Reply:
I tried to use the sampler at first. If so, errors occur as bellow.
```
File "/home/ubuntu/repo/mmlab/PR/mmdetection3d/mmdet3d/datasets/pipelines/dbsampler.py", line 322, in sample_class_v2
    boxes = np.concatenate([gt_bboxes, sp_boxes], axis=0).copy()
  File "<__array_function__ internals>", line 5, in concatenate
ValueError: all the input array dimensions for the concatenation axis must match exactly, but along dimension 1, the array at index 0 has size 7 and the array at index 1 has size 9
```
This error may caused by the misalignment of the size between gt_bboxes and sp_boxes. 
It should be solved in future. But I'm not familiar with the sampler now, so I just commented the config file.

Reply:
You can set the config of db_sampler to load the correct dimensions of gt_boxes.

Reply:
I add `bbox_code_size` attribute for `DataBaseSampler` to solve this problem in the new commit.